dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -216,17 +226,20 @@ mydec_sales AS (
                 + COALESCE(line_10k, 0)
             ) > 0 AS sale_filter_ptax_flag,
             COUNT() OVER (
-                PARTITION BY line_1_primary_pin, line_4_instrument_date
+                PARTITION BY
+                    line_1_primary_pin,
+                    line_4_instrument_date,
+                    line_2_total_parcels
             ) AS num_single_day_sales,
-            year_of_sale
+            year_of_sale AS year
         FROM {{ source('sale', 'mydec') }}
-        WHERE line_2_total_parcels = 1 -- Remove multisales
```

Comment:
Okay, I did a lot of digging today, and this is the source of the current discrepancies between new data where `source = 'iasworld'` and the old data (letting in mydec multisales). it affects how `num_single_day_sales` is built. I *believe* i've fixed this by adding an additional partition to the construction of `num_single_day_sales` and then only excluding pins with more than one sale per day if it isn't a multisale.